src/components/spreadsheet/spreadsheet.ts

```diff
@@ -264,6 +264,9 @@ export class Spreadsheet extends Component<SpreadsheetProps, SpreadsheetChildEnv
     const { xRatio, yRatio } = this.env.model.getters.getFrozenSheetViewRatio(
       this.env.model.getters.getActiveSheetId()
     );
+
+    if (!isFinite(xRatio) || !isFinite(yRatio)) return; // Returning NaN and Infinity before mounted causing warning does not disappear
```

Comment:
minor style detail: we tend to avoid one-line ifs. It's sometimes harder to read and spot the `return`.
I would also slightly rephrase the comment
```suggestion
    if (!isFinite(xRatio) || !isFinite(yRatio)) {
      // before mounting, the ratios can be NaN or Infinity if the viewport size is 0
      return;
    }
```